### ISSUE TITLE
Training RORem-mixed details

### ISSUE THREAD

**Issue by hdjsjyl** [leeruibin/RORem#12]
hi author,
    Thanks for sharing the excellent work.
    Would you share the training details of the model rorem-mixed?
    Are rorem-mixed and rorem using the same training parameters other than input resolution? How about the number of gpus? How about the training steps? How about the learning rate? 

training 1024x1024 was using pretrained model in 512x512 or during training 512x512 and 1024x1024 input are randomly chosen for training?

Thanks
